addons/mail/static/src/core/web/messaging_menu.dark.scss

```diff
@@ -0,0 +1,5 @@
+.o-mail-MessagingMenu-navbar button {
+    &.o-active, &:hover {
+        background-color: mix($o-gray-200, $o-gray-300);
```

Comment:
Can't we settle on a color that we already have?

Reply:
Can't find one that has either similar color or semantics.